File: exercises/day19-North-Pole-Companies/java/src/main/java/com/northpole/Invoice.java
package com.northpole;

import java.util.List;

public class Invoice {
    public final String customer;
    public final List<Delivery> deliveries;

    public Invoice(String customer, List<Delivery> deliveries) {
        this.customer = customer;
        this.deliveries = deliveries;
    }
}

File: exercises/day19-North-Pole-Companies/csharp/NorthPole/Invoice.cs
using System.Collections.Generic;

namespace NorthPole;

public class Invoice
{
    public string Customer { get; }
    public List<Delivery> Deliveries { get; }

    public Invoice(string customer, List<Delivery> deliveries)
    {
        Customer = customer;
        Deliveries = deliveries;
    }
}